assignment4.Rmd

```diff
@@ -54,6 +58,7 @@ D2 <- scale(D2, center = TRUE)
 
 ```{r}
 pca <- prcomp(D2, scale = TRUE)
+names(pca)
```

Comment:
You can also view the objects within an R object in the "environment" tab of RStudio.